core/src/main/java/org/apache/accumulo/core/metrics/MetricsProducer.java

```diff
@@ -617,6 +617,13 @@
   String METRICS_UPDATE_WALOG_WRITE = METRICS_UPDATE_PREFIX + "walog.write";
   String METRICS_UPDATE_MUTATION_ARRAY_SIZE = METRICS_UPDATE_PREFIX + "mutation.arrays.size";
 
+  String METRICS_PROPSTORE_PREFIX = "accumulo.prop.store.";
```

Comment:
Please update the class javadoc with these new properties

Reply:
Fixed in e7dc935faf